jupyterlab_gitsync/src/components/setup.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
```

Comment:
Are these files ready to merge to master? Please don't submit files that are still under work for review. 1) You will not get an approval to merge for the whole PR, because these files are not done yet. 2) It wastes reviewers' time to look into these files and try to understand.

That's why I suggested break the code into sub-components with good description. It helps reviewers understand what you are doing, and they don't need to jump between 40 files to trace implementations of functions. I will be very hesitate to approve this PR, since I'm not sure if everything works fine.

Always be considerate about reviewers' review experience, since they spend their time helping you finish your work.

Reply:
This is just a placeholder file so that the compiler does not complain. I will be doing this implementation later on.